### ISSUE TITLE
Sound not playing on Microsoft Edge

### ISSUE THREAD

**Issue by tehmaestro** [CreateJS/SoundJS#188]
Hi. Sound is not working on Windows 10's Microsoft Edge browser. What I noticed is that, the request is not even send to grab the audio file.
  Also, calling `window.createjs.Sound.getCapabilities()` returns null, so I get this is the problem? Will further investigate ..

**Comment by gskinner** [comment]
Great. I really appreciate you taking a look!

**Comment by wdamien** [comment]
@tehmaestro can you provide a reproduction case?  Testing with the demos at http://createjs.com/soundjs work using Edge 20.10240.16384.0

**Comment by raduca-adrian** [comment]
@wdamien the demos don't seem to work in my Microsoft Edge, using the version you mentioned
![image](https://cloud.githubusercontent.com/assets/2914382/9091844/26c3bad8-3bab-11e5-9694-768a083a4dba.png)

**Comment by wdamien** [comment]
@raduca-adrian; Are you seeing any console errors when running that page?

**Comment by raduca-adrian** [comment]
@wdamien 
Only one, but seems related to the font
HTTP403: FORBIDDEN - The server understood the request, but is refusing to fulfill it.
(XHR): GET - http://use.typekit.net/c/3b54bf/1w;effra,2,ZYf:R:i4,ZYd:R:n4,ZYj:R:n7;proxima-nova,2,b5x:R:i4,b61:R:i7,b5w:R:n4,bBh:R:n7/d?3bb2a6e53c9684ffdc9a9afe1d5b2a6262b4be0bfc491ced4e51babf615367c177f0d9460eb02f0cdecba30fed480ada2cf4f68c86d0fbcd7d4c3539816f4f0efa1b197ded903568ea273a9d779d4f6de02bdb949d814d

**Comment by wdamien** [comment]
Thanks @raduca-adrian ! Yeah I see the same error, it would not affect playback.

We will keep looking into this on our side, its most likely a browser bug with specific systems. If you or @tehmaestro have any more details or insights, please post them here.

**Comment by tehmaestro** [comment]
Hi, so I've further tested my project on a couple of others PC's that were updated to Windows 10, and all seems to be ok. No problems..
  So it appears it would be a PC related issue.

Thanks.

**Comment by digitalmouse** [comment]
In a related note, I just started testing soundjs and howler.js for sound support projects I am assigned to at work.  Everything was working fine until I upgraded to Windows 10 to test Edge, and upgrading to iOS 9 for Apple products.  Prior to that, the sound code working fine with Chrome, Firefox, Opera, IE11, mobile Chrome, and mobile Safari.  Got the iOS stuff working, but MS Edge still gives me trouble.   In my case soundjs is throwing a `SoundJS is not currently supported in your browser.` and linking me to this github tracker.  howler.js fails with a console output of "not supported" and pointing to JavaScript code that reads `audiotest = new Audio()` - which is standard Audio API access.  (howler references added for extra info for debugging sound in general, not bring the issue off-topic).
